mne/decoding/time_gen.py

```diff
@@ -390,11 +446,13 @@ def _predict_time_loop(X, estimators, cv, slices, predict_mode,
     ----------
     X : ndarray, shape (n_epochs, n_features, n_times)
         To-be-fitted data.
-    estimators : array-like, shape (n_times, n_folds)
-        Array of scikit-learn classifiers fitted in cross-validation.
-    slices : list
-        List of slices selecting data from X from which is prediction is
-        generated.
+    estimators : list of array-like, shape (n_times, n_folds)
+        List of array of scikit-learn classifiers fitted in cross-validation.
```

Comment:
I'm not sure to follow. I suggest you open a new PR so that this discussion doesn't get squashed after address your comments. 

Try to come up with a use case, so that it'd be clear that we need to change the current API, and we can perhaps also do a Skype with Jona Denis & Alex at some point.